pkg/deck/deck.go

```diff
@@ -0,0 +1,12 @@
+package deck
+
+import (
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/stack"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tiles"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tilesets"
+)
+
+type Deck struct {
```

Comment:
Isn't this struct a bit redundant? Since both Stack and TIleSet contains complete list of ordered tile

Reply:
It would be enough for this struct to have a `StartingTile` and `Stack`, yes. I felt like having the ability to get the whole `TileSet` from this wrapping structure would be useful. You are right that it is a bit redundant though, I'll switch it to a method that constructs `TileSet` from `StartingTile` and `Stack.GetTiles()`.

Reply:
Done.